source/_integrations/refoss.markdown

```
@@ -67,3 +64,9 @@ Examples:
 ## Reset device
 
 Long-press the button on the device to reset it.
+
+## Known issues
+
+- When prompted: `No devices found on the network`:
+  - Check if the device and Home Assistant are on the same network.
+  - Start Home Assistant in debug mode and add it again, check if any devices are found in the logs,log keyword: `Discovered devices`, If you don't see the device, please check the network and device version.
```

Markdownlint (0.37.0)

Files should end with a single newline character

(MD047, single-trailing-newline)